scala-generator/src/main/scala/models/KafkaConsumer.scala

```diff
@@ -18,6 +19,15 @@ object KafkaConsumer extends CodeGenerator {
     Right(generateCode(form))
   }
 
+  def generateTopicRegex(topicFn: String, apiVersion: String) = {
+    val apiVersionVariable = Seq("${apiVersion}", "$apiVersion").map(Regex.quote(_)).mkString("|")
```

Comment:
Yes we do. What it does here is generating a regex that matches either `${apiVersion}` or `$apiVersion`. Then it's used to replace them with the actual value, e.g. `v1`.

Looking at the code, I think we can drop this bit as there is a val `apiVersion` defined in the same `object`, which means we don't need to do this when generating the code. e.g. this should work:

```
def topic(instance: String)(tenant: String) = s"mc.data.member-internal.$apiVersion.$instance.$tenant"
```